build/webpack/plugins/react-loadable-plugin.js

```diff
@@ -0,0 +1,58 @@
+// Implementation of this PR: https://github.com/jamiebuilds/react-loadable/pull/132
+// Modified to strip out unneeded results for Next's specific use case
+const url = require('url')
+
+function buildManifest (compiler, compilation) {
+  let context = compiler.options.context
+  let manifest = {}
+
+  compilation.chunks.forEach(chunk => {
+    chunk.files.forEach(file => {
+      chunk.forEachModule(module => {
+        let id = module.id
+        let name = typeof module.libIdent === 'function' ? module.libIdent({ context }) : null
+        // If it doesn't end in `.js` Next.js can't handle it right now.
```

Comment:
Why not `.mjs`? Webpack v4 in fact prefers it.

Reply:
This is the output filename, not the input filename.